analyzers/src/SonarAnalyzer.Common/AnalysisContext/SonarCodeBlockAnalysisContext.cs

```diff
@@ -20,9 +20,9 @@
 
 namespace SonarAnalyzer;
 
-public sealed class SonarCodeBlockAnalysisContext : SonarAnalysisContextBase<CodeBlockAnalysisContext>
```

Comment:
Shouldn't rename then the classes to `SonarCodeBlockReportingContext` to be more specific? 

(I fine also with keeping them as they are since the class `SonarReportingContextBase` derives from `SonarAnalysisContextBase`)

Reply:
You're in trouble. I like the idea.. will do it at the end in another PR :)